src/test/java/org/example/PersonTest.java

```diff
@@ -0,0 +1,171 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersonTest {
+
+    @Test
+    void addParents() {
+        Person father = new Person("Jan", "Jong", 40,"male");
+        Person mother = new Person("Ana", "Jong", 40, "female");
+        Person person = new Person("Laura", "Jong", 20,"female");
+
+        person.addParents(mother, father);
+        Person result = person.getFather();
+
+        assertEquals(father, result);
+
+    }
+
+    @Test
+    void childShouldBeAddedToPerson() {
+        Person person = new Person("Laura", "Jong", 50,"female");
+        Person child1 = new Person("Ana", "Jong", 20, "female");
+        Person child2 = new Person("Jan", "Jong", 26,"male");
+
+        person.addChild(child1);
+        person.addChild(child2);
+        Person result = person.getChildren().get(1);
+
+        assertEquals(child2, result);
+
+    }
+
+    @Test
+    void siblingShouldBeAddedToPerson() {
+        Person person = new Person("Laura", "Jong", 45,"female");
+        Person sibling1 = new Person("Ana", "Jong", 20, "female");
+        Person sibling2 = new Person("Jan", "Jong", 26,"male");
+
+        person.addSibling(sibling1);
+        person.addSibling(sibling2);
+        Person result = person.getSiblings().get(1);
+
+        assertEquals(sibling2,result);
+    }
```

Comment:
Vergelijk je hier niet dezelfde input en output? Want eerst doe je addSibling en daarna getSibling, dus logischerwijs komt daar hetzelfde uit? Als je gebruikt maakt van een losse List<Person> sibblings = new ArrayList<>(); kun je die vergelijken met de addSibling methode + getter. Zie uitwerkingen: https://github.com/hogeschoolnovi/backend-java-family-tree-uitwerkingen/blob/main/src/test/java/PersonTest.java

Reply:
addSiblings is de methode, die ik test en ik vergelijk Person sibling2 met person.getSiblings().get(1).